packages/models/src/types.utils.ts

```diff
@@ -11,3 +11,10 @@ export type LiteralUnion<LiteralType, BaseType extends Primitive> =
   | (BaseType & Record<never, never>);
 
 export type Entries<T> = { [K in keyof T]: [K, T[K]] }[keyof T][];
+
+export type ReplaceTypes<T, Replacements extends { [K in keyof T]?: any }> = Omit<
+  T,
+  keyof Replacements
+> & {
+  [K in keyof Replacements]: Replacements[K];
+};
```

Comment:
What does this do?

Reply:
I wanted to reuse part of the @stacks/transactions types but replace their use of `IntegerType` from @stacks/common with our `Money` type and narrow the other types rather than installing another lib to handle their `IntegerType`. I could completely create our own types copied from @stacks/transactions, but I looked up this utility to replace types. Now, we can pass our Money type all the way to the stacks pkg and then just convert it to a string their before using their typed function. Look at use in `transaction.types.ts`.